src/main/resources/assets/moreculling/lang/zh_cn.json

```diff
@@ -6,8 +6,19 @@
 	"moreculling.config.option.blockStateCulling.tooltip": "启用激进的方块状态剔除",
 	"moreculling.config.option.customItemFrameRenderer": "自定义物品展示框渲染器",
 	"moreculling.config.option.customItemFrameRenderer.tooltip": "启用自定义物品展示框渲染器",
-	"moreculling.config.option.itemFrameLOD": "物品展示框多层次细节",
+	"moreculling.config.option.itemFrameLOD": "展示框多层次细节",
 	"moreculling.config.option.itemFrameLOD.tooltip": "启用物品展示框多层次细节",
-	"moreculling.config.option.itemFrame3FaceCulling": "三面物品展示框剔除",
-	"moreculling.config.option.itemFrame3FaceCulling.tooltip": "启用三面剔除技术"
-}
\ No newline at end of file
+	"moreculling.config.option.itemFrameLODRange": "展示框多层次细节范围",
+	"moreculling.config.option.itemFrameLODRange.tooltip": "更改剔除开始运作的距离",
+	"moreculling.config.option.itemFrame3FaceCulling": "三面展示框剔除",
+	"moreculling.config.option.itemFrame3FaceCulling.tooltip": "启用三面剔除技术",
+	"moreculling.config.option.itemFrame3FaceCullingRange": "三面展示框剔除范围",
+	"moreculling.config.option.itemFrame3FaceCullingRange.tooltip": "更改剔除开始运作的距离，在更小的距离上可能可见",
+	"moreculling.config.option.leavesCulling": "树叶剔除",
+	"moreculling.config.option.leavesCulling.tooltip": "更改用于树叶的剔除类型，越高越快！",
```

Comment:
Change the `Higher is faster!` to be in the leavesCullingDepth line. I just changed that, once you do that ill merge it and publish v0.6.0